app/uk/gov/hmrc/thirdpartydeveloperfrontend/helpers/DateFormatter.scala

```diff
@@ -34,13 +34,16 @@ object DateFormatter {
     standardFormatter.print(dateTime)
   }
 
-  def formatLastAccessDate(lastAccessDate: DateTime, createdOnDate: DateTime): Option[String] = {
```

Comment:
Why not just test for None and return "Never accessed" ?

Reply:
or is it "Never used"

Reply:
Because that's a change in logic